### PR TITLE
Added Button

### PR DESCRIPTION
**Pull request by dubeyaditya29**
Added button Component 

https://user-images.githubusercontent.com/48158220/153487727-732bc945-1cf7-4e44-88aa-6e20fe07a083.mp4

### CODE REVIEW

File: components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+.btn {
+  position: relative;
+  font-family: inherit;
+  box-sizing: border-box;
+  cursor: pointer;
+  font-weight: var(--text-bold);
+  color: var(--color-white);
+  outline: auto;
+  border-radius: 0.5rem;
```
Comment (Hamdrive) [inline components/Button/button.css]: Consider using `px` for such small values

File: components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  box-shadow: 3px 3px 0px rgb(119 136 153);
```
Comment (Hamdrive) [inline components/Button/button.css]: Add ` , ` to separate the values inside `rgb ()`

File: components/Button/button.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+           &lt;button class="btn btn-size-sm btn-color-success"&lt;Button&lt;/button&gt;
+          &lt;/a&gt;
+        </pre>
+
+        
+    </main>
+    <script
+      src="https://kit.fontawesome.com/daef01c36c.js"
+      crossorigin="anonymous"
+    ></script>
```
Comment (Hamdrive) [inline components/Button/button.html]: You can import this directly within CSS, by grabbing FontAwesome CDN link from website and pasting in your index.css

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Have given some optional suggestions
Have also requested to fix the wrong syntax in `rgb()`

Overall code looks good and clean

File: components/Button/button.css (hunk 1 above)
Comment (dubeyaditya29) [reply components/Button/button.css]: Have changed the 0.5rem to its relevant value in px.

File: components/Button/button.css (hunk 2 above)
Comment (dubeyaditya29) [reply components/Button/button.css]: Thanks for pointing this out have fixed it.

File: components/Button/button.html (hunk 3 above)
Comment (dubeyaditya29) [reply components/Button/button.html]: Thanks for the input didn't knew this.

**Review by ZuberDunge — APPROVED** [review]
Apart from what hamza suggested everything looks good

File: components/Button/button.html (hunk 3 above)
Comment (dubeyaditya29) [reply components/Button/button.html]: Have imported this link in css file.

**Review by Hamdrive — COMMENTED** [review]
Reviewed updated changes, looks good!
Approved and can be merged to dev!

**Pull request merged**